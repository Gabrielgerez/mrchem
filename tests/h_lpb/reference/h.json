{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "molecule": {
      "cavity": {
        "spheres": [
          {
            "alpha": 1.0,
            "beta": 0.0,
            "center": [
              0.0,
              0.0,
              0.0
            ],
            "radius": 2.645616384,
            "sigma": 0.2
          }
        ]
      },
      "charge": -1,
      "coords": [
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 6,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "h",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.0001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.0001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.0001,
          "shared_memory": false,
          "smooth_prec": 0.0001
        },
        "reaction_operator": {
          "DHS-formulation": "variable",
          "density_type": "total",
          "dynamic_thrs": false,
          "epsilon_in": 1.0,
          "epsilon_out": 78.4,
          "formulation": "exponential",
          "ion_radius": 0.0,
          "ion_width": 0.2,
          "kain": 6,
          "kappa_out": 0.08703231499578493,
          "max_iter": 100,
          "optimizer": "potential",
          "poisson_prec": 0.0001,
          "solver_type": "Linearized_Poisson-Boltzmann"
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (PBE0)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.0001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        0.0
      ],
      "charge": -1,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 8.17913080064801e-13,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            0.0
          ],
          "vector_el": [
            0.0,
            0.0,
            0.0
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          0.04928658666294078
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": 0.09857317332588156
      },
      "scf_energy": {
        "E_ee": 1.220280348193974,
        "E_eext": 0.0,
        "E_el": -0.618896701038129,
        "E_en": -1.9145538948639602,
        "E_kin": 0.9258603759485293,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": 0.10754370176892118,
        "E_tot": -0.5113529992692079,
        "E_x": -0.15252084470528907,
        "E_xc": -0.4885331476130991,
        "Er_el": -0.20942953799828384,
        "Er_nuc": 0.10754370176892118,
        "Er_tot": -0.10188583622936287
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.220280348193974,
        "E_eext": 0.0,
        "E_el": -0.618896701038129,
        "E_en": -1.9145538948639602,
        "E_kin": 0.9258603759485293,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": 0.10754370176892118,
        "E_tot": -0.5113529992692079,
        "E_x": -0.15252084470528907,
        "E_xc": -0.4885331476130991,
        "Er_el": -0.20942953799828384,
        "Er_nuc": 0.10754370176892118,
        "Er_tot": -0.10188583622936287
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
