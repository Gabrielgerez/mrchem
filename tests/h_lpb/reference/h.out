                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         pb-solvation                             ***
***         Git commit hash    0608bccb3e71737b2169                     ***
***         Git commit author  Gabriel Gerez                            ***
***         Git commit date    Tue Oct 3 15:39:21 2023 +0200            ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.5.0-alpha
 Git branch            : HEAD
 Git commit hash       : a3618d1498410124ec47
 Git commit author     : gitpeterwind
 Git commit date       : Tue Feb 21 15:21:15 2023 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (8 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 6
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                              -1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                             Solvation Cavity
---------------------------------------------------------------------------
 Formulation             :                                     exponential
 Dielectric constant     :            (in)                        1.000000
                         :           (out)                       78.400000
---------------------------------------------------------------------------
    N      R_0  Alpha Beta Sigma      Radius         x         y         z
---------------------------------------------------------------------------
    0   2.6456  1.00  0.00  0.20  ->  2.6456  0.000000  0.000000  0.000000
===========================================================================


===========================================================================
              Square of the Debye-Huckel screening parameter
---------------------------------------------------------------------------
 Formulation             :                                     exponential
 Screening function value:            (in)                        0.000000
                         :           (out)                        0.087032
---------------------------------------------------------------------------
    N      R_0 Sigma      Radius         x         y         z
---------------------------------------------------------------------------
    0   2.6456  0.20  ->  2.6456  0.000000  0.000000  0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999999944384e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.925860375949
 E-N energy              :            (au)                 -1.914553894864
 Coulomb energy          :            (au)                  1.220280348194
 Exchange energy         :            (au)                 -0.152520844705
 X-C energy              :            (au)                 -0.488533147613
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Reaction energy (el)    :            (au)                 -0.209429537998
 Reaction energy (nuc)   :            (au)                  0.107543701769
 Reaction energy (tot)   :            (au)                 -0.101885836229
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -0.618896701038
 Nuclear energy          :            (au)                  0.107543701769
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.113529992692e-01
                         :      (kcal/mol)             -3.208788516320e+02
                         :        (kJ/mol)             -1.342557115228e+03
                         :            (eV)             -1.391462397116e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                  0.049286586663
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  0.098573173326
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                              -1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.925860375949
 E-N energy              :            (au)                 -1.914553894864
 Coulomb energy          :            (au)                  1.220280348194
 Exchange energy         :            (au)                 -0.152520844705
 X-C energy              :            (au)                 -0.488533147613
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Reaction energy (el)    :            (au)                 -0.209429537998
 Reaction energy (nuc)   :            (au)                  0.107543701769
 Reaction energy (tot)   :            (au)                 -0.101885836229
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -0.618896701038
 Nuclear energy          :            (au)                  0.107543701769
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.113529992692e-01
                         :      (kcal/mol)             -3.208788516320e+02
                         :        (kJ/mol)             -1.342557115228e+03
                         :            (eV)             -1.391462397116e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                  0.049286586663
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  0.098573173326
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  6s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
